xclim/indices/_anuclim.py

```diff
@@ -191,11 +191,11 @@ def precip_seasonality(
     with xarray.set_options(keep_attrs=True):
         seas = 100 * _anuclim_coeff_var(pr)
 
-    seas.attrs["units"] = "percent"
+    seas.attrs["units"] = "%"
```

Comment:
Are you sure this works ? % gave me trouble in the past.

Reply:
Well you are kinda right, I realize my tests worked because you (or someone else) already put the workaround in `units2pint`. I can put "percent".  I tested and cfunits recognizes both '%' and 'percent', but not 'pct'.